pkg/shares/split_sparse_shares_test.go

```diff
@@ -38,3 +39,38 @@ func TestSparseShareSplitter(t *testing.T) {
 	got := sss.Export()
 	assert.Len(t, got, 2)
 }
+
+func TestWriteNamespacePaddingShares(t *testing.T) {
+	ns1 := appns.MustNewV0(bytes.Repeat([]byte{1}, appns.NamespaceVersionZeroIDSize))
+	blob1 := newBlob(ns1, appconsts.ShareVersionZero)
```

Comment:
could we use a different more arbitrary share version here without breaking anything? The reasoning being that the test as is would have passed before this change, but different share versions wouldn't have.

Reply:
I attempted to but it fails for a different reason earlier in the test case b/c every other share version is unsupported.

Reply:
TBH I'm unsatisfied too b/c I want a test case with a different share version.

Reply:
```
	t.Run("with a different share version", func(t *testing.T) {
		ns2 := appns.MustNewV0(bytes.Repeat([]byte{2}, appns.NamespaceVersionZeroIDSize))
		blob1 := newBlob(ns2, 1)

		sss := NewSparseShareSplitter()

		err := sss.Write(blob1)
		assert.NoError(t, err)
	})
```

```
--- FAIL: TestWriteNamespacePaddingShares (0.00s)
    --- FAIL: TestWriteNamespacePaddingShares/with_a_different_share_version (0.00s)
        /Users/rootulp/git/rootulp/celestia/celestia-app/pkg/shares/split_sparse_shares_test.go:75: 
            	Error Trace:	/Users/rootulp/git/rootulp/celestia/celestia-app/pkg/shares/split_sparse_shares_test.go:75
            	Error:      	Received unexpected error:
            	            	unsupported share version: 1
            	Test:       	TestWriteNamespacePaddingShares/with_a_different_share_version
        /Users/rootulp/git/rootulp/celestia/celestia-app/pkg/shares/split_sparse_shares_test.go:77: 
            	Error Trace:	/Users/rootulp/git/rootulp/celestia/celestia-app/pkg/shares/split_sparse_shares_test.go:77
            	Error:      	Received unexpected error:
            	            	cannot write namespace padding shares on an empty SparseShareSplitter
            	Test:       	TestWriteNamespacePaddingShares/with_a_different_share_version
FAIL
FAIL	github.com/celestiaorg/celestia-app/pkg/shares	1.208s
FAIL
```